packages/ember-routing/lib/system/route.js

```diff
@@ -1616,7 +1616,9 @@ var Route = EmberObject.extend(ActionHandler, Evented, {
     needs access to a parent route's model in order to
     resolve a model (or just reuse the model from a parent),
     it can call `this.modelFor(theNameOfParentRoute)` to
-    retrieve it.
+    retrieve it. Returns an object, not a promise, 
```

Comment:
I think it's important it says it won't return a promise, as that's the confusing part for beginners: knowing whether you'll get the plain data object or something you should/can call then on.